src/brogue/Rogue.h

```diff
@@ -1283,12 +1283,22 @@ enum dungeonLayers {
     NUMBER_TERRAIN_LAYERS
 };
 
+typedef struct CellTextInfo {
+    /// 0 : Normal terminal output
+    /// 1 : Left-aligned text [startColumn must be set]
+    /// 2 : Middle-aligned text [startColumn and endColumn must be set]
+    int8_t mode;
```

Comment:
Q: what about an enum?

Reply:
Yep, that would be a good idea, I was being lazy.

I'll fix this after I rebase the PR to fix the conflicts.